core-bundle/src/EventListener/RequestTokenListener.php

```diff
@@ -62,13 +68,18 @@ public function onKernelRequest(GetResponseEvent $event): void
 
         // Only check the request token if a) the request is a POST request, b)
         // the request is not an Ajax request, c) the _token_check attribute is
-        // not false and d) the _token_check attribute is set or the request is
-        // a Contao request
+        // not false, d) the _token_check attribute is set or the request is a
+        // Contao request and e) the request has cookies, a session or is authenticated
```

Comment:
`aka it even requires CSRF protection`.

Reply:
15aafd2b32a07b143e52b7ae2d9d1d0953664af8